doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
-
-Here is the step by step process to generate code:
+#### 2. dotnet new project
+- create project folder 'Azure.<group>.<service>'. e.g. Azure.IoT.DeviceUpdate under sdk/<service> folder
+- new project: navigate to the project folder, and run 'dotnet new' as following:
+  
+```
+sdk\<your-service-name>\Azure.<group>.<service>> dotnet new dataplane --libraryName [Client-Library-Title] --swagger [input-swagger-file-path] --securityScopes [security-scopes] --force
+```
+e.g.
+```
+dotnet new dataplane --libraryName DeviceUpdate --swagger https://github.com/Azure/azure-rest-api-specs/blob/23dc68e5b20a0e49dd3443a4ab177d9f2fcc4c2b/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2021-06-01-preview/deviceupdate.json --securityScopes https://api.adu.microsoft.com/.default --force
+```
 
-#### 1. Set the Swagger File
+#### 3. update the solution file if needed
+Run 'dotnet sln' as following to update the projects in the solution file:
 
-Open autorest.md in your `Azure.<group>.<service>` project. In the input-file definition, replace the file path with your swagger link, for example: [WebPubSub swagger link](https://github.com/Azure/azure-rest-api-specs/blob/39c7d63c21b9a29efe3907d9b949d1c77b021907/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json). Update a namespace definition to set the namespace C# generator will use when creating your types. Update the AAD token credential(for AAD token, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/src/autorest.md#autorest-configuration) example) or Azure key credential (for Azure Key credential, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/autorest.md#swagger-sources) example). Save your changes.
+```
+dotnet sln remove src\Azure.<group>.<service>.csproj
+dotnet sln add src\Azure.<group>.<service>.csproj
+dotnet sln remove tests\Azure.<group>.<service>.Tests.csproj
+dotnet sln add tests\Azure.<group>.<service>.Tests.csproj
+```
 
-Your `autorest.md` file should now look like:
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
 
-````md
-# `Azure.<group>.<service>` Code Generation
+### Generate client library
 
-Run `dotnet build /t:GenerateCode` to generate code.
+In this section, we'll create a generated API layer built on Azure Core.
 
-### AutoRest Configuration
-> see https://aka.ms/autorest
+From a PowerShell command prompt, navigate to the directory holding `Azure.<group>.<service>.csproj`. Run the following commands:
 
-``` yaml
-input-file:
-- <swagger-file-link>
-namespace: <namespace>
-public-clients: true
-data-plane: true
-security: AADToken
-security-scopes: <AAD-token-scope>
 ```
-````
+sdk\<your-service-name>\Azure.<group>.<service>\src> dotnet build /t:GenerateCode
+```
```

Comment:
Why don't we invoke autorest directly?  This isn't consistent with the way we generate Python. I would like the processes to as similar as possible across our Tier1 languages.

Reply:
.Net integrate autorest call into dotnet build target. It is the new approach, and go seems accept the same solution.

Reply:
@mikekistler, the autorest call is quite complex - we have this nice tool that simplifies it for us and that our partners can benefit from as well.

Reply:
@annelo-msft Will this tool be implemented to support all the Dataplane Codegen languages? We should not have five different ways of generating DataPlane Codegen libraries.  Ideally the instructions would vary only the parameter passed to autorest to say which language to generate.  I know that's idealistic, but maybe that's the PM's job ;-).

Reply:
@mikekistler, I respectfully disagree that all languages should adopt a twenty(ish) argument command line command in favor of a three word one for the sake of consistency.  Maybe we could think about what the ideal command line would be and find a way for all languages to move toward consistency that way, so we wouldn't have to trade off usability for consistency?